tasks/meta.yml

```diff
@@ -56,4 +53,20 @@
     line: "connInterfacesFile           = /etc/beegfs/connInterfacesFile"
   become: true
   notify: Restart BeeGFS meta service
+
+- name: Modify tuneTargetChooser settings
```

Comment:
Can we make this into a dict to iterate over? e.g.
beegfs_meta_conf:
    tuneTargetChooser: x
    tuneNumWorkers: y

then:

- name: Modify tuneNumWorkers settings
  lineinfile:
    path: /etc/beegfs/beegfs-meta.conf
    regexp: "^{{ item.key }}"
    line: "{{ item.key }}               = {{ item.value }}"
  become: true
  with_items: beegfs_meta_conf

This opens up the possiblity of introducing a similar pattern for other conf files later.

Also adding this to the readme would be nice.

Reply:
Good suggesstion. Will add the changes